code/modules/customitems/item_defines.dm

```diff
@@ -654,6 +655,24 @@ All custom items with worn sprites must follow the contained sprite system: http
 	else
 		..()
 
+/obj/item/fluff/tokash_spear/MouseDrop(mob/user as mob)
+	if((user == usr && (!( usr.restrained() ) && (!( usr.stat ) && (usr.contents.Find(src) || in_range(src, usr))))))
+		if(!istype(usr, /mob/living/carbon/slime) && !istype(usr, /mob/living/simple_animal))
```

Comment:
kinda niche checks. check if the mob is human (i.e. a character) instead.

Reply:
I pulled this MouseDrop code from  the paperbin code.
Aurora.3/code/modules/paperwork/paperbin.dm

I'm not sure how I would rewrite it to make a different chcek.
Possibly if(istype(usr, /mob/living/carbon/human) ?